robotoff/models.py

```diff
@@ -146,6 +152,22 @@ def create_from_latent(cls, latent_insight: "ProductInsight", **kwargs):
         return cls.create(**updated_values)
 
 
+class AnnotationVote(BaseModel):
+    id = peewee.UUIDField(primary_key=True, default=uuid.uuid4)
```

Comment:
I tried to not even provide `id`, but it didn't work - and since I'm more used to autoincrementing ints for PKs, I thought this was the way you do it with UUIDs. What is the correct way?

Reply:
I'm not sure if it's a peewee issue or a postgres issue, but there indeed does not seem to be a way to just leave it unset, without a default, and have an ID auto-assigned.

I saw an alternative way to have it auto-assigned with   `peewee.UUIDField(.. ,constraints=[SQL("DEFAULT uuid_generate_v4()")])`

but that also was throwing errors for me..

Reply:
Let's leave it like this then. Feel free to reopen the conversation if you disagree :D